frontend/src/component/operator-wizard/OperatorWizardFooter.js

```diff
@@ -27,13 +29,24 @@ export default class OperatorWizardFooter extends React.Component {
                 Next
               </Button>
             );
+            const onViewYamlBtn = () => {
```

Comment:
If the "onViewYamlBtn" is a event handler, please consider rename it to something like "onViewYamlBtnClick", since multiple event handlers may be binded to one element. For example, we may have "onViewYamlBtnHover" to handle the hover event of the button.

Reply:
right now there is only one event, but yes renaming to "..BtnClick" is a good idea.